HOWTO.md

```diff
@@ -0,0 +1,73 @@
+# How to create serverless applications using AWS SAM
```

Comment:
I wonder if this is the place to discuss what a stack is? I noticed we discussed it below, but didnt quite define what it is. I like your below definition of what a changeset is, but maybe a simple definition of a stack here would be helpful